listenbrainz/webserver/static/js/src/stats/__snapshots__/UserEntity.test.tsx.snap

```diff
@@ -15,105 +15,139 @@ exports[`User Artists Page renders 1`] = `
       className="row"
     >
       <div
-        className="col-md-4"
+        className="col-xs-12"
       >
         <h3>
-          Top Artists
+          Top
+          <span
+            className="dropdown"
+          >
+            <button
+              className="dropdown-togle btn-transparent"
+              data-toggle="dropdown"
+              style={
+                Object {
+                  "fontWeight": "bold",
+                  "textTransform": "capitalize",
+                }
+              }
```

Comment:
Maybe it would be worth to put this in one of the less stylesheets and use a class

Reply:
Done